### PR TITLE
add eslint config to repo to integrate with text editors

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

**Comment by origamiserviceuser** [comment]
o-grid bundle size difference from 5.1.0 to 1a65a7b79fd947e950363a65a1b12876a14d300c
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
o-grid bundle size difference from 5.1.0 to 7a3a4a8e75c1c56c0221b2c6809dd6ee629f981b
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
o-grid bundle size difference from 5.1.0 to fa9f6f7483089e819e72569fc63078761134564b
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
o-grid bundle size difference from 5.1.0 to d1210c0483e163366d7286a47c0149ccdcc239f1
No significant bundle size differences found.

File: demos/src/js/style-switcher.js [hunk 1]
```diff
@@ -53,8 +53,8 @@ const local = /localhost|0\.0\.0\.0/.test(document.URL);
 
 function convertKeywordsToSpans(keyword) {
 	// If it's a number, return it directly
-	if (keyword * 1 >= 0) {
-		return keyword * 1;
+	if (Number(keyword) >= 0) {
```
Comment (notlee) [inline demos/src/js/style-switcher.js]: better

**Comment by origamiserviceuser** [comment]
o-grid bundle size difference from 5.1.0 to a7a7352ff38c3b3be681b32024d63d9c1811ed6c
No significant bundle size differences found.

**Pull request merged**